docs/root/_static/css/envoy.css

```diff
@@ -46,6 +46,11 @@ table.docutils div.line-block {
     border: solid #eee 1px;
 }
 
+/* restore margin bottom on aligned images */
+.rst-content img.align-center {
+    margin-bottom: 24px
+}
+
```

Comment:
nit: seems this is additional view change?

Reply:
yeah - a small css fix to give images (that have been aligned) some space at the bottom, consistent with if they had not been aligned